UDPServer/MainWindow.xaml.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace UDPServer
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+
+        private readonly MyUDPServer server = new MyUDPServer();
+        bool portValid;
+        bool ipValid;
+        CancellationTokenSource ctSource;
+        CancellationToken token;
+
+        public MainWindow()
+        {
+            portValid = false;
+            ipValid = false;
+            InitializeComponent();
+            //ctSource = new CancellationTokenSource();
```

Comment:
Please don't leave commented code behind. It will confuse yourself and people who code after you.

Reply:
Removed.